tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrderBuilder.cs

```diff
@@ -1,131 +1,172 @@
-﻿using System;
-using NHSD.BuyingCatalogue.Ordering.Domain;
+﻿﻿using System;
+ using System.Collections.Generic;
+ using NHSD.BuyingCatalogue.Ordering.Domain;
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders
 {
     internal sealed class OrderBuilder
     {
-        private readonly Order _order;
+        private string _orderId;
+        private string _orderDescription;
+        private Guid _organisationId;
+        private readonly string _organisationName;
```

Comment:
Also seen as there is a OrderBuilder and OrderItemBuilder in the domain layer why do we need one here?

Reply:
Both the domain and api layers use the domain classes for test data. It would make sense to move these builders into a common library.